packaging/installer/kickstart.sh

```diff
@@ -311,10 +346,17 @@ dependencies
 # ---------------------------------------------------------------------------------------------------------------------
 # download netdata package
 
-set_tarball_urls "${RELEASE_CHANNEL}"
+if [ -z "${NETDATA_LOCAL_TARBALL_OVERRIDE}" ]; then
+	set_tarball_urls "${RELEASE_CHANNEL}"
+
+	download "${NETDATA_TARBALL_CHECKSUM_URL}" "${TMPDIR}/sha256sum.txt"
+	download "${NETDATA_TARBALL_URL}" "${TMPDIR}/netdata-latest.tar.gz"
+else
+	progress "Installation sources were given as input, running installation using \"${NETDATA_LOCAL_TARBALL_OVERRIDE}\""
+	run mv "${NETDATA_LOCAL_TARBALL_OVERRIDE_CHECKSUM}" "${TMPDIR}/sha256sum.txt"
```

Comment:
Maybe `cp` instead of `mv`ing and `rm` if the installation succeeds. This way, if there is a problem with the install, the user won't have to re-copy the files to `${TMPDIR}`. Typically files in `${TMPDIR}` will be cleared after sometime anyway (when depends on operating system/linux distro etc.) 

What do you think?

Reply:
this is indeed a great catch.
as agreed on slack i 'll copy the files, but not remove them in the end since these are what the user downloaded and he may want to keep them around